cfgov/v1/tests/test_wagtail_hooks.py

```diff
@@ -206,6 +207,28 @@ def test_tag_param_returns_resources_tagged_that_way(self):
         )
 
 
+@unittest.skipIf(wagtail.VERSION < (2, 0), "No need to test in Wagtail 1+")
+class TestWhitelistOverride(SimpleTestCase):
+    # Borrowed from https://github.com/wagtail/wagtail/blob/master/wagtail
+    # /core/tests/test_whitelist.py
+
+    def test_whitelist_hooks(self):
+        """Test that Whitelister does not strip scan element and attributes.
+
+        The allowed element <scan> and attributes are added in v1.wagtail_hooks.
+        """
+        input_html = '''
+<span class="schema-container"
+      itemprop="step"
+      itemscope=""
+      itemtype="http://schema.org/HowToSection">
+    <span class="schema-container" itemprop="itemListElement">
+    </span>
+</span>
+        '''
+        output_html = Whitelister.clean(input_html)
+        self.assertHTMLEqual(input_html, output_html)
+
```

Comment:
Craig, I might be missing something, but this test seems duplicative of the one just below it.

Reply:
The content of `input_html` is different in `test_whitelist_hooks` depending on `if wagtail.VERSION < (2, 0)` or `if wagtail.VERSION > (2, 0)`.
`input_html` contains only `<span>` for Wagtail 2.x

Reply:
Ah, of course, now I see! Thanks for explaining.